PWGHF/DataModel/DerivedTables.h

```diff
@@ -564,7 +564,7 @@ DECLARE_SOA_TABLE_STAGED(HfBplusMcs, "HFBPMC", //! Table with MC candidate info
 // normalised: DecayLengthNormalised, DecayLengthXYNormalised, ImpactParameterNormalised0
 DECLARE_SOA_TABLE_STAGED(HfLcPars, "HFLCPAR", //! Table with candidate properties used for selection
                          hf_cand::Chi2PCA,
-                         hf_cand::NProngsContributorsPV,
+                         hf_cand::IndicesProngsContributorsPV,
```

Comment:
Hi @vkucera,
My idea was that indices of prongs which contribute into PV is more general (full) information which can be saved in the same size field. Therefore I propose to replace an existing column (because having two of them will be redundant) - NProngsContributorsPV can be easily evaluated on the fly from IndicesProngsContributorsPV.

Reply:
The problem I see is that you are removing a value which is used directly also outside O2Physics in postprocessing and you are replacing it with a value which needs to be interpreted with a helper function inside O2Physics. Adding the dynamic column does not help because it won't exist when reading the tables outside O2Physics.

Reply:
So if I return back the `NProngsContributorsPV` column as a usual, non-dynamic, will it be a good solution?

Reply:
I think you should first clarify where exactly in the code you plan to use the new column.
In the current version of the PR, you remove `NProngsContributorsPV` from two workflows (`derivedDataCreatorLcToPKPi.cxx`, `treeCreatorLcToPKPi.cxx`) where it must stay and you don't use the new column `IndicesProngsContributorsPV` anywhere else.

Reply:
I plane to use it in post-processing tasks, the input for which is prepared by `treeCreatorLcToPKPi.cxx`, so in the O2Physics code I don't use the new column indeed.

Reply:
OK, in that case I think the best is to have both columns in `treeCreatorLcToPKPi.cxx` without changing derived data model.

Reply:
I changed the PR in order to preserve the previously existing `NProngsContributorsPV` column and not to modify those parts of code where it's not needed for my purposes.

Reply:
Thanks, the description is updated now.